pom.xml

```diff
@@ -100,7 +100,7 @@
 			<plugin>
 				<groupId>org.codehaus.mojo</groupId>
 				<artifactId>exec-maven-plugin</artifactId>
-				<version>3.1.0</version>
+				<version>3.3.0</version>
```

Comment:
keep this 3.1, also take latest pull, I had updated fixed this issue wit version